[FIX] hr_employee_firstname: fix DeprecationWarning

edy1192 opened the pull request:
Run the README generator to fix the warning message in the log:

`DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, hr_employee_firstname can just have a UTF8 description with not need for a declaration.`

luisg123v commented on hr_employee_firstname/static/description/index.html:
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```
This file is autogenerated.

Could you check if that line is removed by running the README generator, please?

You can read instructions on how to run it [here](https://github.com/OCA/maintainer-tools/?tab=readme-ov-file#readme-generator).

edy1192 replied:
Thanks! Done. Could you check it, please?

edy1192 committed:
[FIX] hr_employee_firstname: fix DeprecationWarning

Run the README generator to fix the warning message in the log:

    DeprecationWarning: XML declarations in HTML module
    descriptions are deprecated since Odoo 17,
    hr_employee_firstname can just have a UTF8 description
    with not need for a declaration.

luisg123v reviewed (approved):
LGTM :+1:

luisg123v commented:
/ocabot merge patch
